build.gradle.kts

```diff
@@ -48,7 +48,7 @@ subprojects {
     version = rootProject.version
     group = rootProject.group
 
-    val kotlinApiVersion = org.jetbrains.kotlin.gradle.dsl.KotlinVersion.KOTLIN_1_4
+    val kotlinApiVersion = org.jetbrains.kotlin.gradle.dsl.KotlinVersion.KOTLIN_1_6
```

Comment:
This breaks the compatibility with MPS versions before 2022.1. See https://plugins.jetbrains.com/docs/intellij/using-kotlin.html#kotlin-standard-library

Reply:
I've reverted it in [bac7b9a](https://github.com/modelix/modelix.mps-plugins/pull/66/commits/bac7b9aa436db8c877f9ba90a11811b857fcd768).